src/testing.c

```diff
@@ -41,8 +42,16 @@ void testing(){
     printf(COLOR);printf("+----------------------+\n");printf(RESET);
     //Testing sort algorithm for all 3 array of 1.1 
     bubblesort(input, 8);
+
+    **(input)=-1000;  //Added for testing purposes
+    if (sortcheck(input,8) != 0) {
+        printf(COLOR);printf("\nProblems in result of sorting algorithm found...\n\n");printf(RESET);
+        //exit(EXIT_SUCCESS);
```

Comment:
an der stelle sollten wir im Programm dann natürlich mehr als nur eine Ausgabe in der Konsole machen - ggf. fragen ob Abbruch erfolgen soll o.ä.